lang/vi/validation.php

```diff
@@ -0,0 +1,217 @@
+<?php
+
+declare(strict_types=1);
+
+return [
+	'accepted' => 'Trường :attribute phải được chấp nhận.',
```

Comment:
```suggestion
    'accepted' => 'Trường :attribute phải được chấp nhận.',
```
indent 4 spaces, not tab